code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -48,16 +48,23 @@
 	fire = 80
 	acid = 50
 
+/obj/machinery/portable_atmospherics/canister/get_save_vars()
+	. = ..()
+	. += NAMEOF(src, valve_open)
+	. += NAMEOF(src, release_pressure)
+	return .
+
 /obj/machinery/portable_atmospherics/canister/Initialize(mapload, datum/gas_mixture/existing_mixture)
 	. = ..()
 
 	if(mapload)
 		internal_cell = new /obj/item/stock_parts/power_store/cell/high(src)
 
-	if(existing_mixture)
-		air_contents.copy_from(existing_mixture)
-	else
-		create_gas()
+	if(!initial_gas_mix)
+		if(existing_mixture) // this arg has never been used anywhere
+			air_contents.copy_from(existing_mixture)
+		else
+			create_gas()
```

Comment:
didn't I merge a pr of yours getting rid of this

Reply:
You approved (but not merged) the PR in:
- #91196

So whichever PR gets merged first will cause a merge conflict that I will go resolve. Initially, I came across the dead code while working on this and thought it would be better to split it off instead of bundling it here.

Reply:
Applied via https://github.com/tgstation/tgstation/pull/91051/commits/dbb29fd92d67d2babdc30d6d5053f23c009e4d16